src/recommendations/install-configure.md

```diff
@@ -0,0 +1,48 @@
+---
+group: product-recommendations
+title: Install and Configure Recommendations
+redirect_from:
+  - /recommendations/configure.html
+ee_only: True
+---
+
+Deploying Product Recommendations to your storefront requires that you install, configure, then [create the recommendations in the Admin UI](https://docs.magento.com/m2/ee/user_guide/marketing/create-new-rec.html).
+
+## Install Product Recommendations {#install}
+
+1. Install the `magento/product-recommendations` module with Composer:
+
+   ```bash
+   composer require magento/product-recommendations
+   ```
+
+   The `magento/product-recommendations` module requires the following dependencies:
+
+   -  **module-data-services** — This module enables behavioral data collection by tracking [user events on the page]({{ page.baseurl }}/recommendations/events.html). This type of data is required by Adobe Sensei to compute product affinities based on production shopper behavior like product views, products added to a cart, and checkouts. Adobe Sensei then uses this information to create and train machine learning models for each website and storeview. This unlocks recommendation types like "Customers who viewed this, also viewed...", which automatically adjusts with shopper behavior over time. Magento and Adobe Sensei do not collect personally identifiable information.
+
+   -  **saas-export** — This module syncs catalog data. This type of data provides product information to the Product Recommendations service so it can accurately return product names, pricing, images, URLs, inventory and availability, and other attributes.
+
+       {:.bs-callout-info}
+       If you prefer, you can install the above modules explicitly using Composer: `composer require magento/module-data-services` and `composer require magento/saas-export`
+
+   -  Other modules responsible for configuration
+
+### Update your Product Recommendations installation
+
+To update the `product-recommendations` module, run the following:
+
+```bash
+composer update magento/product-recommendations --with-dependencies
+```
+
+### Uninstall Product Recommendations
+
+If needed, you can [uninstall the product-recommendations module]({{ site.baseurl }}/guides/v{{ site.version }}/install-gde/install/cli/install-cli-uninstall-mods.html).
+
+## Configure Product Recommendations {#configure}
+
+After you install the `magento/product-recommendations` module, you must link the module to Magento's SaaS Services by [retrieving the API Key and specifying the SaaS Environment](https://docs.magento.com/m2/ce/user_guide/configuration/services/saas.html).
```

Comment:
can we simplify by saying

> you must configure the module by specifying the API Key and selecting a SaaS Environment

I think I'd avoid using the "linking" verbiage here (it's fine in the userguide intro)